ux.symfony.com/assets/controllers/theme-switch-controller.js

```diff
@@ -0,0 +1,25 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
```

Comment:
We already have https://github.com/symfony/ux/blob/2.x/ux.symfony.com/assets/controllers/theme-switcher-controller.js

Reply:
As i said, it's 100% independant for now, until we know what we do ... it was a lot of work to bring it in, so i wait we decide to polish things :))